assets/overlays/openstack-manila/patches/node_add_driver.yaml

```diff
@@ -57,6 +57,12 @@ spec:
             - name: MANILA_SHARE_PROTO
               value: NFS
           volumeMounts:
+            - name: config-manilaplugin
+              mountPath: /etc/kubernetes/config
+              readOnly: true
```

Comment:
Do you want this? We don't have support for a `cloud.conf` file in the Manila driver yet, right?